src/components/commonComponent.js

```diff
@@ -0,0 +1,15 @@
+export class CommonComponent {
```

Comment:
change common component logic from querySelector to createElement and append this to parent element (parent element from arguments)